src/HomeKit/HMMutablePresenceEvent.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using XamCore.ObjCRuntime;
+
+namespace XamCore.HomeKit {
+
+	partial class HMMutablePresenceEvent {
+
+		public HMPresenceType PresenceType {
```

Comment:
make it `virtual` so we can replace this with _normal_ bindings (once support is added to the generator)